src/styles/components/_dropdown.scss

```diff
@@ -0,0 +1,4 @@
+.c-dropdown__content-open {
+	// Move the open dropdown above the tabs component
+	z-index: 11;
```

Comment:
We have a function `get-z-layer` to get a z-index. In `_variables.scss` you can see the options.

Reply:
i don't see that function. Maybe you're thinking of hetarchief instead of avo?